src/hotspot/share/gc/shenandoah/shenandoahOldGeneration.cpp

```diff
@@ -245,12 +246,12 @@ bool ShenandoahOldGeneration::contains(ShenandoahHeapRegion* region) const {
 }
 
 void ShenandoahOldGeneration::parallel_heap_region_iterate(ShenandoahHeapRegionClosure* cl) {
-  ShenandoahGenerationRegionClosure<OLD> old_regions(cl);
+  ShenandoahIncludeRegionClosure<OLD_GENERATION> old_regions(cl);
```

Comment:
Subjective nit: Not something you introduced, but I'd name the variable `old_region_cl` rather than `old_regions` just to avoid dimensional mismatch (to borrow a term from physics) in the naming. It makes code easier to read if one avoids such naming type/dimension mismatch.

Ignore this suggestion if this will affect upstream Shenandoah as well (unless the renaming can be pushed separately and avoid an issue with the diffs).